proto/buf/alpha/registry/v1alpha1/audit_event.proto

```diff
@@ -0,0 +1,96 @@
+// Copyright 2020-2022 Buf Technologies, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package buf.alpha.registry.v1alpha1;
+
+import "google/protobuf/timestamp.proto";
+
+// AuditActorType is the type of actor that triggered certain audited event.
+enum AuditActorType {
+  // AUDIT_ACTOR_TYPE_UNSPECIFIED is the default actor type for audited events.
+  AUDIT_ACTOR_TYPE_UNSPECIFIED = 0;
+  // AUDIT_ACTOR_TYPE_USER is the actor type for when the audited event is
+  // triggered by a user. This is the most common case.
+  AUDIT_ACTOR_TYPE_USER = 1;
+}
+
+// AuditActor is the actor who triggered or executed the audited event.
+message AuditActor {
+  // Type of actor who caused the audited event.
+  AuditActorType type = 1;
+  // Id of the actor who eecuted the action.
+  string id = 2;
+}
+
+// AuditResourceType is the type of the resource that was affected by the
+// audited event.
+enum AuditResourceType {
+  // AUDIT_RESOURCE_TYPE_UNSPECIFIED is the default type for audited events.
+  AUDIT_RESOURCE_TYPE_UNSPECIFIED = 0;
+  // AUDIT_RESOURCE_TYPE_USER is the type for when the audited event affected a
+  // user.
```

Comment:
complete nit: I might just nix these comments as they aren't adding much on top of the field names :-)

Reply:
Agree, although that would leave the BSR docs w/out any description. Not sure if that's acceptable or if there's an unwritten rule of always adding comments to proto files. Would you remove just the "UNSPECIFIED" comments? or all of them?

Reply:
I'd probably remove all; just a quick look at some of our other enums leads me to believe that we generally don't comment each value and instead generally just comment the enum itself. (See https://github.com/bufbuild/buf/blob/main/proto/buf/alpha/registry/v1alpha1/role.proto, for example).

Admittedly I haven't seen best practices documented around this, but I think this is pretty in-line with what we're doing currently. Definitely subjective, though.

Reply:
Removed